sdk/textanalytics/azure-ai-textanalytics/src/samples/README.md

```diff
@@ -82,6 +88,9 @@ This project welcomes contributions and suggestions. Find [more contributing][SD
 [async_sample_entities]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/RecognizeEntitiesAsync.java
 [async_sample_entities_batch]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/RecognizeEntitiesBatchDocumentsAsync.java
 [async_sample_entities_batch_convenience]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/RecognizeEntitiesBatchStringDocumentsAsync.java
+[async_sample_pii_entities]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/RecognizePiiEntitiesAsync.java
```

Comment:
can't pass ci because of broken link, so will add them back after the sample merged.

https://dev.azure.com/azure-sdk/public/_build/results?buildId=501818&view=logs&j=36df4387-3551-57fa-098c-b43a23930261&t=3851d55a-4253-57f5-016e-927e8de9d21b

Reply:
#14274